app/globals.css

```
@@ -1140,3 +1140,395 @@
+
+.madhava-bubble--user {
+  align-self: flex-end;
+}
+
+.madhava-bubble__content {
+  padding: 10px 14px;
+  border-radius: 12px;
+  font-size: 13px;
+  line-height: 1.55;
+  font-weight: 500;
+  word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)